bgpd/bgp_evpn_vty.c

```diff
@@ -3139,8 +3140,15 @@ static void evpn_show_all_routes(struct vty *vty, struct bgp *bgp, int type,
 		prefix_rd2str((struct prefix_rd *)rd_destp, rd_str,
 			      sizeof(rd_str), bgp->asnotation);
 
-		if (json)
+		if (json) {
+			if (first) {
+				vty_out(vty, "\"%s\":", rd_str);
```

Comment:
I'm confused by these literal json outputs: why do we want to hard-code these kinds of components in the output? if we're not managing the json object hierarchy correctly, I'd rather we fix that than stick some hard-coded tokens in the output like this.

Reply:
Some of the parameters were identified to stringify them in the output so that the overall memory consumption is reduced when the json command was executed in a high scale setup.